libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -116,7 +117,7 @@ void AP_IOMCU::thread_main(void)
         if (mask & EVENT_MASK(IOEVENT_INIT)) {
```

Comment:
Do you need to mask out IOEVENT_SEND_PWM_OUT here?
```mask &= ~EVENT_MASK(IOEVENT_SEND_PWM_OUT);```

Reply:
I didn't mask it as it is a periodic event, but you're right that this means we'll resend it unnecessarily if a later msg fails. I'll add it

Reply:
fixed